buildconfig/appveyor.yml

```diff
@@ -168,8 +168,9 @@ install:
   - "%PYTHON% -m pip install Sphinx==4.5.0 sphinxcontrib-applehelp==1.0.2 pip install sphinxcontrib-devhelp==1.0.2 sphinxcontrib-htmlhelp==2.0.0 sphinxcontrib-qthelp==1.0.3 sphinxcontrib-serializinghtml==1.1.5"
   - "%PYTHON% setup.py docs"
   - "%WITH_COMPILER% %PYTHON% -m buildconfig"
-  - "%WITH_COMPILER% %PYTHON% setup.py build -j4 %USE_ANALYZE%"
-  - "%WITH_COMPILER% %PYTHON% setup.py -setuptools %DISTRIBUTIONS%"
+  # - "%WITH_COMPILER% %PYTHON% setup.py build -j4 %USE_ANALYZE%"
```

Comment:
This is used to turn on the compiler static analysis. I guess there will be some new way to pass flags through or extend the compiler?